jwst/resample/gwcs_drizzle.py

```diff
@@ -199,34 +191,6 @@ def add_image(self, insci, inwcs, inwht=None, xmin=0, xmax=0, ymin=0, ymax=0,
                   xmin=xmin, xmax=xmax, ymin=ymin, ymax=ymax,
                   pixfrac=self.pixfrac, kernel=self.kernel, fillval=self.fillval)
 
-    def blot_image(self, blotwcs, interp='poly5', sinscl=1.0):
```

Comment:
I'm assuming we don't need this function any more?

Reply:
Yeah, it's not used at all, and it's incorrect for `jwst` which uses `gwcs.WCS` objects.  I think it was just copied over from `drizzle` accidentally.

The only time we use `blot` is in `outlier_detection`, and that calls the C blot routine directly, not through this function.  Clearly a refactor in the future would be desirable, but I could say that about many aspects of `resample`.  😅